ci-operator/config/netobserv/netobserv-cluster-operator-main.yaml

```diff
@@ -0,0 +1,88 @@
+base_images:
+  base:
+    name: ubi-minimal
+    namespace: ocp
+    tag: "9"
+  cli:
+    name: "4.13"
+    namespace: ocp
+    tag: cli
+  ocp_builder_rhel-9-golang-1.19-openshift-4.13:
+    name: builder
+    namespace: ocp
+    tag: rhel-9-golang-1.19-openshift-4.13
+binary_build_commands: make build
+build_root:
+  image_stream_tag:
+    name: builder
+    namespace: ocp
+    tag: rhel-9-golang-1.19-openshift-4.13
+images:
+- dockerfile_path: build/Dockerfile
+  from: base
+  inputs:
+    bin:
+      paths:
+      - destination_dir: .
+        source_path: /go/bin/network-observability-operator
```

Comment:
have you been able to test the build or not?
I'm not sure how these configs/steps all play together, but if the build command is `make build` (as shown L14) and this dockerfile uses that build output, then the source should be something like `./bin/manager`

In case it is useful, here's how the build definition was done previously: https://github.com/openshift/release/pull/35766/files#diff-df4f4f7896d9a6f03e1d041f559f8d1b076995b41b84ff5853eb619950b53da3L29-L32
As you can see we also had to override a couple of go ENV variables, but maybe this isn't useful here, as the base image used for building is different.

Reply:
> have you been able to test the build or not?
No, I have not yet, wanted to create this first draft of config and have it pass by you to resolve any evident issues as you noted.

> I'm not sure how these configs/steps all play together
I can't say, I know them well either, trying to go [through the documentation](https://docs.ci.openshift.org/docs/how-tos/testing-operator-sdk-operators/) and use references from other operator configs.

>  as the base image used for building is different.
do you mean `9.1` in Dockerfile vs `9` in this config?

added feedback in commit: [1284cbd](https://github.com/openshift/release/pull/37301/commits/1284cbd929f273c5597c4321d11974837e1572b5)